phpunit.xml.dist

```diff
@@ -11,8 +11,6 @@
          stopOnFailure="false"
          stopOnIncomplete="false"
          stopOnSkipped="false"
-         syntaxCheck="true"
-         strict="true"
```

Comment:
As I've said earlier (I do remember my all comments even though you seem to have force pushed, which erased my previous review results), we need to replace `strict="true"` with the corresponding `beStrict...` options instead of just switching to non-strict mode.

Reply:
does that make a difference?

Reply:
The `strict` option was split out into the several`beStrict...` options and some of them are enabled by default. This way you have to figure out:

1. which of `beStrict...` options are disabled by default
2. would benefit we'll gain by enabling them
3. enable appropriate options

I'll trust you on this.